package-lock.json

```diff
@@ -27135,6 +27136,7 @@
         "@storybook/react": "^7.3.2",
         "@storybook/react-webpack5": "^7.3.2",
         "chromatic": "^7.1.0",
+        "framer-motion": "*",
```

Comment:
Did you intend to keep this one floating?

Reply:
Definitely not 😅 Thanks for catching this! Fixed in [regenerate lock file](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7878/commits/a5f3af468a4ceca1f87cc3cfd900866c90cbcfd6)